prowler/lib/check/check.py

```diff
@@ -716,10 +715,29 @@ def execute(
 
         # Mutelist findings
         if hasattr(global_provider, "mutelist") and global_provider.mutelist:
-            check_findings = mutelist_findings(
-                global_provider,
-                check_findings,
-            )
+            is_finding_muted_args = {}
+            # TODO: make this prettier
+            if global_provider.type == "aws":
+                is_finding_muted_args["aws_account_id"] = (
+                    global_provider.identity.account
+                )
+            elif global_provider.type == "azure":
+                is_finding_muted_args["subscription"] = global_provider.identity.account
+
+            elif global_provider.type == "gcp":
+                is_finding_muted_args["aws_account_id"] = (
+                    global_provider.identity.account
```

Comment:
We have to get here the finding.project_id value

Reply:
We can get it in the `is_finding_muted` function.

Reply:
Great idea!